src/test/functional/functional-tests.js

```diff
@@ -275,7 +318,25 @@ describe('functional tests', function() {
         .catch(done)
     })
 
-    it('should download 100KB Buffer, as promise', done => {
+    step('getPartialObject(bucketName, objectName, offset, length, cb)_offset:1024, length=1024_', done => {
+      client.getPartialObject(bucketName, _100kbObjectBufferName, 1024, 1024)
+        .then(stream => {
+          stream.on('data', function() {})
+          stream.on('end', done)
+        })
+        .catch(done)
+    })
+
+    step('getPartialObject(bucketName, objectName, offset, length, cb)_offset:1024', done => {
+      client.getPartialObject(bucketName, _100kbObjectBufferName, 1024)
+        .then(stream => {
+          stream.on('data', function() {})
+          stream.on('end', done)
```

Comment:
Would it be beneficial here to check the partial object that is returned, kind of like how it checks the hash sum elsewhere?